src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -5739,6 +5739,27 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
   </Target>
 
+  <!--
+    ***********************************************************************************************
+    ***********************************************************************************************
+                                                                Test Section
+    ***********************************************************************************************
+    ***********************************************************************************************
+  -->
+  <!--
+    ============================================================
+    This stub `VSTest` target allows for targets implementing Test execution functionality
+    to run after it.
+    
+    For example:
+    <Target Name="RunVSTest" AfterTargets="VSTest">
+     (implementation)
+    </Target>
+    ============================================================
+  -->
+
+  <Target Name="VSTest"></Target>
```

Comment:
Why did we change? VS[Test] is VisualStudio prerix, I think that [MSBuild] prefix is better

Reply:
It was requested by [nohwnd](https://github.com/nohwnd) and [Evangelink](https://github.com/Evangelink)

Reply:
How about MSBuildVSTest ?

Reply:
I don't think we can use the name "Test" for the target for compatibility reasons.

Reply:
`Test` is way too common target name (mostly being used for prototyping use-cases "let's try this MSBuild code - so let's name it foo or test"). 

See e.g. https://grep.app/search?q=%3CTarget%20Name%3D%22Test%22

`MSBuildRunTests` sounds great. It doesn't seem to be used "in the wild" and it would not conflict with anything built-in

Reply:
The problem with `MSBuildRunTests` is that maybe will be used not only to "run" tests it's more an hook/entry point for any kind of test suite so I was wondering if we're able to find a more general purpose name.

Reply:
Also, although I understand your point, `Test` seems more natural and similar to other default targets: `dotnet build -t:Test`.

Reply:
We can't use `Test` - there is plenty of existing usage [in the wild](https://github.com/search?q=%3CTarget+Name%3D%22Test%22+(path%3A*.props+OR+path%3A*.targets+OR+path%3A*.*proj)&type=code) showing that name already in use.

Reply:
That's why @MarcoRossignoli was suggesting the condition that would allow to opt-in/out of the msbuild infra while keeping the nice target name. Obviously, if this is possible to achieve somtthing like that.

Reply:
@JanKrivanek @novacole in case of native aot compilation this target is in the correct position? I mean if I want to get the output and run the native aot generated single module.